src/components/Person.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  partnerName,
+  sex,
+  isMarried,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    <p className="Person__age">
+      {age && `I am ${age}`}
+    </p>
+    <p className="Person__partner">
+      {isMarried
+        ? `My ${sex === 'f'
+          ? `husband's`
+          : `wife's`} name is ${partnerName}`
+        : `I am not married`}
+    </p>
```

Comment:
1. This logic is too complex for JSX - move it to a variable.
2. Don't ever use nested ternary operators.